docs/intro/tool-fundamentals/terragrunt.md

```diff
@@ -0,0 +1,36 @@
+---
+'title': 'Terragrunt'
+---
```

Comment:
You don't need to explicitly have a frontmatter with title if there's a primary header `#` with the exact same wording. Docusaurus will automatically pull the first main header as the title.

Reply:
This applies to other files too.

Reply:
Done in a97edf5